post_office/template/__init__.py

```diff
@@ -0,0 +1,15 @@
+from django.template.loader import get_template, select_template
+
+
+def render_to_string(template_name, context=None, request=None, using=None):
```

Comment:
Is this function only used in tests? If so, please put this in `tests/utils.py`

Reply:
This function is the counterpart of `django.template.engine.render_to_string` for our special template engine. It is intended to be used by those who want to render the final message themselves and later compose their message.
Therefore I would keep it.

Reply:
Ok.